File: src/LPII/Atividade1/Exercicio2.java
package LPII.Atividade1;
import java.util.Scanner;

public class Exercicio2 {
    public static void main(String[] args) {
        Scanner ler = new Scanner(System.in);

        int[] vet1 =  new int[10];
        int[] vet2 = new int[10];
        int[] mult = new int[10];
        int i;

        System.out.println("Digite os 10 primeiros numeros");

        for(i=0; i<10; i++){
            System.out.println("Digite os numeros do 1 vetor");
            vet1[i] = ler.nextInt();
        }

        for(i=0; i<10; i++){
            System.out.println("Digite os numeros do 2 vetor");
            vet2[i] = ler.nextInt();
        }

        for(i=0; i<10; i++) {
            mult[i] = (vet1[i] * vet2[i]);
            System.out.println("Multiplicao dos vetores = " + mult[i]);
        }

    }
}

File: src/LPII/Atividade1/Exercicio2.c
package LPII.Atividade1;

public class Exercicio2 {
    #include <stdio.h>

    int main(){

        int vet1[10], vet2[10], mult[10] ,cont;

        printf("Digite os 10 primeiros numeros\n \n");

        for(cont=0; cont<10; cont++){
            printf("Digite o %d numero do 1 vetor\n", cont+1);
            scanf("%d", &vet1[cont]);}

        for(cont=0; cont<10; cont++){
            printf("Digite o %d numero do 2 vetor\n", cont+1);
            scanf("%d", &vet2[cont]);}

        for(cont=0; cont<10; cont++){
            mult[cont]= (vet1[cont]*vet2[cont]);
            printf("\n A multiplicacao entre %d e %d = %d\n", vet1[cont], vet2[cont], mult[cont]);}

        return 0;
    }
}
